plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -369,6 +383,7 @@ func MapPodsToMachines(lister algorithm.PodLister) (map[string][]*api.Pod, error
 	if err != nil {
 		return map[string][]*api.Pod{}, err
 	}
+	pods = filterNonRunningPods(pods)
```

Comment:
Should this be pushed down into the lister? the lister already ignores some things, IIRC.